We don't currently provide a python package for nvidia-cuda-runtime
to satisfy python package dependencies. The runtime libs are installed
as system apks. Avoid pip-check failures due to this.

diff --git a/TensorRT/python/packaging/libs_wheel/setup.py b/TensorRT/python/packaging/libs_wheel/setup.py
index 054c565b..963f8d49 100644
--- a/TensorRT/python/packaging/libs_wheel/setup.py
+++ b/TensorRT/python/packaging/libs_wheel/setup.py
@@ -23,8 +23,7 @@ import_package_name = "##TENSORRT_MODULE##_libs"
 
 
 def get_requirements():
-    reqs = ["nvidia-cuda-runtime-cu##CUDA_MAJOR##"]
-    return reqs
+    return []
 
 
 setup(
